jetty-jmh/src/main/java/org/eclipse/jetty/http/jmh/MultiPartBenchmark.java

```diff
@@ -177,7 +178,7 @@ public static void setupTrial() throws Exception
     public long testLargeGenerated() throws Exception
     {
         Path multipartRawFile = _file.toPath();
-        Path outputDir = new File("/tmp").toPath();
+        Path outputDir = Paths.get("/dev/null");// Files.createTempDirectory( "jetty_multipart_benchmark");
```

Comment:
do we need this comment after the line

Reply:
just testing the /dev/null with our load server